lib/Cake/Test/Case/View/Helper/FormHelperTest.php

```diff
@@ -10648,4 +10648,42 @@ public function testInputDefaults() {
 		$this->assertEquals($expected, $result);
 	}
 
+/**
+ * Tests `_lastAction`.
+ *
+ *  With named, numeric value
+ *
+ * @return void
+ */
+	public function testLastActionWithNamedNumeric() {
+		$here = '/users/index/page:1';
+
+		$this->Form->request->here = $here;
+		$this->Form->create('User');
+
+		$expected = $here;
+		$actual = $this->Form->_lastAction;
```

Comment:
@markstory Are you sure that `$this->assertAttributeEquals` works as expected, all tests pass now without applying a fix